Sources/BranchSDK/BranchEvent.m

```diff
@@ -134,8 +134,11 @@ - (instancetype)initWithCoder:(NSCoder *)decoder {
     self = [super initWithCoder:decoder];
 	if (!self) return self;
 
-	self.serverURL = [decoder decodeObjectOfClass:NSString.class forKey:@"serverURL"];
-	self.eventDictionary = [decoder decodeObjectOfClass:NSDictionary.class forKey:@"eventDictionary"];
+    self.serverURL = [decoder decodeObjectOfClass:NSURL.class forKey:@"serverURL"];
```

Comment:
Since this failed before, and will succeed now, will link replay be a significant behavior change?

Reply:
It's an edge case error handling change. I've reached out to product about it.

Reply:
product would prefer to fix/test it